src/Mod/Drawing/Gui/TaskOrthoViews.cpp

```diff
@@ -252,7 +250,6 @@ void orthoview::set_projection(const gp_Ax2& cs)
 
     calcCentre();
 
-    //this_view->Direction.setValue(Z_dir.X(), Z_dir.Y(), Z_dir.Z());
```

Comment:
In principal OK, but has nothing to do with msc.

Please only make changes related to MSC in this PR.